test/CMakeLists.txt

```
@@ -14,6 +14,12 @@ target_link_libraries(test_dictionary PRIVATE Catch2::Catch2 NeoFOAM
 
 list(APPEND CMAKE_MODULE_PATH ${catch2_SOURCE_DIR}/extras)
 
+add_executable(test_ParallelAlgorithms "test_ParallelAlgorithms.cpp"
+                                       "test_main.cpp")
+target_link_libraries(test_ParallelAlgorithms PRIVATE Catch2::Catch2 NeoFOAM
+                                                      Kokkos::kokkos)
+
 add_test(NAME field_test COMMAND test_field)
 add_test(NAME executor_test COMMAND test_executor)
 add_test(NAME dictionary_test COMMAND test_dictionary)
+add_test(NAME parallelAlgorithms_test COMMAND test_ParallelAlgorithms)
```

using should work for now until we merge https://github.com/exasim-project/NeoFOAM/pull/82/files
```suggestion
neofoamunittest(parallelAlgorithms)
```

---

See https://github.com/exasim-project/NeoFOAM/blob/main/test/core/CMakeLists.txt